mail_tracking/models/mail_thread.py

```diff
@@ -1,17 +1,26 @@
 # Copyright 2019 Alexandre Díaz
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
-from odoo import models, api, _
+from odoo import fields, models, api, _
 from email.utils import getaddresses
 from odoo.tools import email_split_and_format
+from lxml import etree
 
 
 class MailThread(models.AbstractModel):
     _inherit = "mail.thread"
 
+    failed_message_ids = fields.One2many(
+        'mail.message', 'res_id', string='Failed Messages',
+        domain=lambda self:
+            [('model', '=', self._name)]
+            + self.env['mail.message']._get_failed_message_domain(),
+        auto_join=True)
+
     @api.multi
     @api.returns('self', lambda value: value.id)
     def message_post(self, *args, **kwargs):
+        """ Store email_cc in mail.message """
```

Comment:
```suggestion
        """Store email_cc in mail.message."""
```

This weird spacing is a tendency I see and I don't quite understand... :thinking: 

You can review [PEP 257 on the single-line docstrings section](https://www.python.org/dev/peps/pep-0257/#one-line-docstrings). It's not there in any example.

The same for all single-line docstrings here and there.